front/style/pages/_dashboard.scss

```diff
@@ -0,0 +1,97 @@
+.dashboard {
+    margin-bottom: 25px;
+    a {
+        text-decoration: none;
+    }
+
+    .newblue {
+        color: $blue--dark;
+    }
+
+    .button {
+        width: 199px;
+        height: 47px;
+        border: solid 1px #000000;
+        background-color: #ffffff;
+        padding: 10px;
+        margin: 10px 0;
+        text-transform: uppercase;
+    }
+
+    .dashboard_tag {
```

Comment:
Can you use `-` instead of underscore here ?

Reply:
done